R/subscript-tables.R

```diff
@@ -17,10 +17,20 @@
     # Throw a nicer error if the indexing is not appropriate
     if (n.index.args != 1 && n.dim != n.index.args)
         throwErrorTableIndexInvalid(input.name, x.dim)
-    y <- NextMethod(x)
+    # class(x) <- class(x)[!class(x) %in% "qTable"]
+    # y <- `[`(x, ..., drop = drop)
+    y <- NextMethod(`[`, x)
     called.args <- as.list(called.args[["..."]])
+
+    ## Need to evaluate the arguments here to alleviate possible NSE issues; c.f.:
+    ## http://adv-r.had.co.nz/Computing-on-the-language.html#calling-from-another-function
+    evaluated.args <- called.args
+    for (i in seq_along(called.args))
+        if (!identical(as.character(called.args[[i]]), ""))
```

Comment:
Could check for empty arg instead of convert to character? (in the same vein as the `isEmptyList` function?